contracts/Cross-chain/OmnichainProposalSender.sol

```diff
@@ -204,7 +216,8 @@ contract OmnichainProposalSender is ReentrancyGuard, BaseOmnichainControllerSrc
      * @param to_ Address of the receiver
      * @param pId_ The proposal ID to identify a failed message
      * @param remoteChainId_ The LayerZero id of the remote chain
-     * @param payload_ The payload to be sent to the remote chain. It's computed as follows: payload = abi.encode(abi.encode(targets, values, signatures, calldatas), pId)
+     * @param payload_ The payload to be sent to the remote chain.
```

Comment:
Just a small tip: let's not use any full stops anywhere from netspec since we're not using them anywhere.